app.ts

```diff
@@ -0,0 +1,17 @@
+import express, { Application } from "express";
+import dotenv from "dotenv";
+import cookieParser from "cookie-parser";
+
+dotenv.config();
+const app: Application = express();
+
+app.use(cookieParser());
+app.use(express.json());
+
+app.listen(process.env.PORT, () => {
+  console.log(`Server running on port ${process.env.PORT}`);
+});
+
+// 라우터 임포트
+
+// 라우터 설정
```

Comment:
아래와 같은 느낌으로 할일에 대한 주석은 표현해주시면 좋을 것 같아요!
```suggestion
// @TODO 라우터 임포트

// @TODO 라우터 설정
```